src/Functional.Unions.FluentAssertions/Functional.Unions.FluentAssertions/Extensions/OptionExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Functional.Unions.FluentAssertions.Extensions
+{
+	internal static class OptionExtensions
+	{
+		public static T ValueUnsafe<T>(this Option<T> source)
```

Comment:
The latest `Functional.Primitives.Extensions` package [defines a `ThrowOnNone` extension method](https://github.com/JohannesMoersch/Functional/pull/106) (2.4.0)

Reply:
Nice addition! I will use that instead.